vignettes/phlyoVision.Rmd

```diff
@@ -0,0 +1,79 @@
+---
+title: "VisCas Vignette"
+author: "Yanay Rosen"
+date: "9/30/2020"
+output: html_document
```

Comment:
What's the difference between the document `metastasisPhyloVision` and this one `phyloVision.Rmd`? 

I think we only need one vignette showcasing Phylovision.

Reply:
phyloVision.rmd replicates chan 2019 example from Hotspot paper, metastasisPhyloVision is for the lg4 and lg7 trees, I included it for replicability for the paper but can remove!